src/styles/catalog.scss

```diff
@@ -0,0 +1,26 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 48px 46px;
+  justify-content: center;
+  align-items: center;
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
```

Comment:
You are using media queries to change the number of columns for different screen sizes, which is great. However, you are hardcoding the column width (200px) in each media query. It would be more flexible if you use `1fr` instead of `200px` so the grid items will take up the remaining space evenly. This way, the grid will be more responsive and will look good on all screen sizes. Here is a suggestion: 

scss
@media (min-width: 488px) {
  .catalog {
    grid-template-columns: repeat(2, 1fr);
  }
}

@media (min-width: 768px) {
  .catalog {
    grid-template-columns: repeat(3, 1fr);
  }
}

@media (min-width: 1024px) {
  .catalog {
    grid-template-columns: repeat(4, 1fr);
  }
}
```

This is a recommendation for improving the responsiveness of your layout.